wsd/COOLWSD.cpp

```diff
@@ -1439,11 +1439,8 @@ void COOLWSD::innerInitialize(Poco::Util::Application& self)
         if (ConfigUtil::getConfigValue<bool>(conf, "logging.anonymize.allow_logging_user_data",
                                              false))
         {
-            LOG_WRN("Enabling trace logging while anonymization is enabled due to logging.anonymize.allow_logging_user_data setting. "
-                    "This will leak user-data!");
-
-            // Disable anonymization as it's useless now.
-            AnonymizeUserData = false;
```

Comment:
Fair enough. I added description.

(This whole PR is moving the Anonymization globals into the class, so this was natural to delete. But you are right, it might not be obvious when the patch is viewed in isolation.)